### DISCUSSION TITLE
Questions about the education certification of Chinese students

### DISCUSSION THREAD

**Discussion by sIeep1ng** [community/community#52220] [GitHub Education]
I uploaded my student status verification report from Xuexin.com and told me that I could not verify my academic affiliation through this

**Comment by samus-aran** [comment]
Hi @Charlotteicecream  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Comment by whitewatercn** [comment]
可以参考我的这个帖子再逝一逝 https://xlog.whitewater.wang/Github_Student_Developer_Pack
You can refer to this post to try again https://xlog.whitewater.wang/Github_Student_Developer_Pack
